cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -1,74 +1,92 @@
-+
-[source,terminal]
-----
-$ odo delete --deploy
-----
-// #5237
-* Adds mapping support to the `odo link` command
-// #5279
-* Supports ephemeral volumes using the `ephemeral` field in `volume` components
-// #5270
-* Sets the default answer to `yes` when asking for telemetry opt-in
-// #5260
-* Improves metrics by sending additional telemetry data to the devfile registry
-// #5287
-* Updates the bootstrap image to `registry.access.redhat.com/ocp-tools-4/odo-init-container-rhel8:1.1.11`
-// #5308
-* The upstream repository is available at link:https://github.com/redhat-developer/odo[]
+== Notable changes and improvements in odo version 3.0
 
+`odo` 3.0 includes new commands to improve the developer experience. These commands provide the same functionality as before, but in a faster and more efficient manner.
 
+For more information, see the section "Major highlights of odo 3.0" in xref:./understanding-odo.adoc#understanding-odo[Understanding odo].
 
 [id="odo-fixed-issues_{context}"]
-== Bug fixes
-// #5294
-* Previously, `odo deploy` would fail if the `.odo/env` file did not exist. The command now creates the `.odo/env` file if required.
-// #5286
-* Previously, interactive component creation using the `odo create` command would fail if disconnect from the cluster. This issue is fixed in the latest release.
+== Bug fixes in odo version 3.0
 
+* Before this update, `odo` did not honor directory entries in the `.gitignore` file, for example, `/target`. With this update, `odo` honors all `.gitignore` entries. 
+link:https://github.com/redhat-developer/odo/issues/4389[(odo-4389)]
 
-[id="odo-getting-support_{context}"]
-== Getting support
+* Before this update, port names did not match endpoint names. With this update, `odo` ensures that the names match.
+link:https://github.com/redhat-developer/odo/issues/4737[(odo-4737)]
 
-.For Product
+* Before this update, when running the `odo deploy` command, variables defined in a devfile would not be substituted in URI-referenced deployment manifest YAML files. With this update, `odo` performs the variable substitution correctly.
+link:https://github.com/redhat-developer/odo/issues/5451[(odo-5451)]
 
-If you find an error, encounter a bug, or have suggestions for improving the functionality of `{odo-title}`, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose *OpenShift Developer Tools and Services* as a product type and *odo* as a component.
+* Before this update, running the command `odo init --name <name> --devfile-path <path>` caused a panic to occur with a non-existent path. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5540[(odo-5540)]
 
-Provide as many details in the issue description as possible.
+* Before this update, `odo` commands ignored namespace settings in the `.odo/env/env.yaml` file. With this update, `odo` applies the namespace setting.
```

Comment:
```suggestion
* Before this update, `odo` commands ignored namespace settings in the `.odo/env/env.yaml` file. With this update, running the `odo` command applies the namespace setting.
```

Reply:
there is no singular `odo` command - this is referring to the `odo` product that applies the namespace for all odo commands so I think it is ok as is

Reply:
ack